Packs/ProofpointTAP/Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py

```diff
@@ -444,7 +616,589 @@ def fetch_incidents(client, last_run, first_fetch_time, event_type_filter, threa
+                                                            'threatTime', 'campaignID'],
+                                                   headerTransform=pascalToSpace)
+
+    readable_output = messages_readable_output + "\n" + threats_info_readable_output
+
+    if request_error:
+        readable_output += "\n" + tableToMarkdown('Errors',
+                                                  request_error, headers=['interval', 'message'],
+                                                  headerTransform=pascalToSpace
+                                                  )
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'Proofpoint.Messages{messages_type.capitalize()}',
+        outputs=outputs,
+        outputs_key_field=['GUID', 'id'],
+        raw_response=raw_responses
+    )
+
+
+def list_campaigns_command(client: Client, interval: str = None, limit: str = None, page: str = None,
+                           time_range: str = None) -> CommandResults:
+    """
+    Retrieves a list of IDs of campaigns active in a time window.
+    Args:
+        client (Client): ProofpointTAP API client.
+        interval (str): ISO8601-formatted interval date. The minimum interval is thirty seconds. The maximum interval is one day.
+        limit (str): The maximum number of campaign IDs to produce in the response.
+        page (str): The page of results to return, in multiples of the specified size.
+        time_range (str): Time range, for example: 1 week, 2 days, 3 hours etc.
+    Returns:
+        CommandResults: raw response, outputs, and readable outputs.
+    """
+
+    if not (interval or time_range):
+        raise Exception('Must provide interval or time_range.')
+    if interval and time_range:
+        raise Exception('Must provide only one of the arguments interval or time_range.')
+    if time_range and dateparser.parse("7 days") > dateparser.parse(time_range):
+        raise Exception('The maximum time range is 7 days')
+    if time_range and dateparser.parse("30 seconds") < dateparser.parse(time_range):
+        raise Exception('The minimum time range is thirty seconds.')
+
+    if time_range and dateparser.parse("1 hour") < dateparser.parse(time_range):
+        end = datetime.utcnow().strftime(DATE_FORMAT)
+        start = dateparser.parse(time_range).strftime(DATE_FORMAT)
+        intervals = [f'{start}/{end}']
+    else:
+        intervals = handle_interval(dateparser.parse(time_range), is_days_interval=True) if time_range else [
+            interval]  # type: ignore
+
+    outputs = []
+    raw_responses = []
+    request_error = []
+    for interval_string in intervals:
+        try:
+            raw_response = client.list_campaigns(interval_string, page, limit)
+        except ValueError:
+            request_error.append(
+                {'interval': interval_string, "message": f'Not found campaigns data from {interval_string}'})
+            continue
```

Comment:
I agree with you, but the main reason I referred to such a case is because of the 'list-campaigns' command (the current function).
When there is an API request with an interval where there are no campaigns - an error is thrown. The error is thrown from the implementation of the 'http_request' function that existed in the previous integration (the reason is that the API returns 404 status - 'Not found', and therefore the error is thrown).
Would you suggest keep this reference only in the 'list_campaigns_command' function?